netlink-packet-route/src/rtnl/constants.rs

```diff
@@ -394,6 +394,11 @@ pub const IFLA_INFO_DATA: u16 = 2;
 pub const IFLA_INFO_XSTATS: u16 = 3;
 pub const IFLA_INFO_SLAVE_KIND: u16 = 4;
 pub const IFLA_INFO_SLAVE_DATA: u16 = 5;
+// Bridge flags
+pub const IFLA_BRIDGE_FLAGS: u16 = 47;
```

Comment:
I really wanted this to be `0` but code has implemented `IFLA_BR_XXXX` for bridge which blocks `0`.
I think `IFLA_BR_XXXXX` should be implemented for a general link not for `InfoBridge` but trying not to break what is already there.

Reply:
Sorry I don't understand what you mean. Are you saying `IFLA_BRIDGE_FLAGS` can't be named `IFLA_BR_FLAGS`?